File: device/fido/enclave/metrics.cc
// Copyright 2024 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

#include "device/fido/enclave/metrics.h"

#include "base/metrics/histogram_functions.h"

namespace device::enclave {

void RecordEvent(Event event) {
  base::UmaHistogramEnumeration("WebAuthentication.EnclaveEvent", event);
}

}  // namespace device::enclave

File: chrome/browser/resources/chromeos/accessibility/switch_access/mv3/metrics.ts
// Copyright 2017 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

import {StringUtil} from '/common/string_util.js';

/**
 * Class to record metrics for Switch Access.
 */
export namespace SwitchAccessMetrics {
  export function recordMenuAction(menuAction: string): void {
    const metricName = 'Accessibility.CrosSwitchAccess.MenuAction.' +
        StringUtil.toUpperCamelCase(menuAction);
    chrome.metricsPrivate.recordUserAction(metricName);
  }
}
